### PR TITLE
Redirect to "Options" config path more accurate (in Currency Rate form)

### LINKED ISSUE #29336: [Issue] Redirect to "Options" config path more accurate (in Currency Rate form)
This issue is automatically created based on existing pull request: magento/magento2#29276: Redirect to "Options" config path more accurate (in Currency Rate form)

Redirect to "Currency Option Path" more accurate (Auto open the collapse)

### Preconditions (*)
1. Magento 2.4

### Steps to reproduce (*)

1. Go to backend, Stores->Currency Rates
2. Click "Options"

image

### Expected result (*)
Should be redirect to currency options in System config and automatically open the collapse like that:
image

### Actual result (*)
(Now, it is not opened)
image

### PR DESCRIPTION
**Pull request by edenduong**
### Description (*)

Redirect to "Currency Option Path" more accurate (Auto open the collapse)

### Manual testing scenarios (*)

1. Go to backend, Stores->Currency Rates
2. Click "Options"

image

=> Should be redirect to currency options in System config and automatically open the collapse like that:

image

(Now, it is not opened)

image

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#29336: Redirect to "Options" config path more accurate (in Currency Rate form)

### CODE REVIEW

**Comment by rogyar** [comment]
Hi @edenduong. Great improvement thank you.
More kudos for the unit test, good job. 
If you could create a separate MFTF test for this case that will check the menu item expanding behavior, you would get even more kudos :)
But if you don't wish to, it's OK. 

Thank you.

File: app/code/Magento/CurrencySymbol/Block/Adminhtml/System/Currency.php [hunk 1]
```diff
@@ -41,7 +41,14 @@ protected function _prepareLayout()
...
 
-        $onClick = "setLocation('" . $this->getUrl('adminhtml/system_config/edit/section/currency') . "')";
+        $currencyOptionPath = $this->getUrl(
+            'adminhtml/system_config/edit',
+            [
+                'section' => 'currency',
+                '_fragment' => 'currency_options-link'
+            ]
+        );
+        $onClick = "setLocation('" . $currencyOptionPath . "')";
```
Comment (rogyar) [inline app/code/Magento/CurrencySymbol/Block/Adminhtml/System/Currency.php]: You may simplify this statement slightly

```suggestion
        $onClick = "setLocation('$currencyOptionPath')";
```

Comment (edenduong) [reply app/code/Magento/CurrencySymbol/Block/Adminhtml/System/Currency.php]: Thank you for your suggestion :)

**Review by gabrieldagama — APPROVED** [review]
Hi @edenduong, thanks for the pull request, it looks great! The changes are approved.

**Comment by gabrieldagama** [comment]
@magento run all tests

**Comment by magento-engcom-team** [comment]
Hi @gabrieldagama, thank you for the review.
[ENGCOM-8066](https://jira.corp.magento.com/browse/ENGCOM-8066) has been created to process this Pull Request

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-8066](https://jira.corp.magento.com/browse/ENGCOM-8066) has been created to process this Pull Request

**Comment by engcom-Echo** [comment]
Hi, @edenduong. I will take care of test coverage.

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Manual testing scenario:**

1. Go to backend, Stores->Currency Rates;
2. Click "Options";

**Before:** :heavy_multiplication_x: redirected to currency options in System config but not collapsed the tab

Peek 2020-09-04 09-38

**After:** :heavy_check_mark: redirected to currency options in System config and automatically opened the collapse

Peek 2020-09-04 09-49

**Comment by engcom-Echo** [comment]
@magento run all tests

**Pull request merged**